nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -379,6 +379,26 @@ protected Document getChild(String parentId, String name) {
         }
     }
 
+    protected List<String> getChildrenIds(String parentId, boolean excludeSpecialChildren) {
```

Comment:
There's lots of redundancy with the original implementation that doesn't have the `excludeSpecialChildren` parameter.